core/src/main/java/com/linecorp/armeria/client/retry/Backoff.java

```diff
@@ -58,6 +58,53 @@ static Backoff exponential(long initialDelayMillis, long maxDelayMillis, double
     }
 
     /**
+     * Returns a {@link Backoff} that computes backoff delay which is a random value between
+     * {@code minDelayMillis} and {@code maxDelayMillis} chosen by {@link ThreadLocalRandom}.
+     */
+    static Backoff random(long minDelayMillis, long maxDelayMillis) {
+        return random(minDelayMillis, maxDelayMillis, ThreadLocalRandom::current);
+    }
+
+    /**
+     * Returns a {@link Backoff} that computes backoff delay which is a random value between
+     * {@code minDelayMillis} and {@code maxDelayMillis}.
+     */
+    static Backoff random(long minDelayMillis, long maxDelayMillis, Supplier<Random> randomSupplier) {
+        return new RandomBackoff(minDelayMillis, maxDelayMillis, randomSupplier);
+    }
+
+    /**
+     * Creates a new {@link Backoff} that computes backoff delay using one of
+     * {@link #exponential(long, long, double)}, {@link #fixed(long)} and {@link #random(long, long)} chaining
+     * with {@link #withJitter(double, double)} and {@link #withMaxAttempts(int)} from the
+     * {@code specification}. This is the format for the {@code specification}:
+     * <ul>
+     *     <li>{@code exponential=[initialDelayMillis:maxDelayMillis:multiplier]}
+     *     is for {@link Backoff#exponential(long, long, double)}
+     *     (maxDelayMillis and multiplier can be omitted)</li>
```

Comment:
- Please use 2-space indentation for HTML tags.
- Please indent the text in a tag:

  ```html
  <ul>
    <li>first line
        second line</li>
  </ul>
  ```